CHANGELOG

```diff
@@ -1,6 +1,20 @@
 devel
 -----
 
+* Added new metric: "arangodb_sequential_collection_locks" this will count
```

Comment:
We already have some collection lock-related metrics, but they follow a different naming scheme:
* arangodb_collection_lock_timeouts_exclusive
* arangodb_collection_lock_timeouts_write
* arangodb_collection_lock_acquisition_micros
* arangodb_collection_lock_acquisition_time
To make naming consistent, the name of the new lock should be `arangodb`collection_lock_sequential_mode` or such.

Reply:
Thank you you are right.
Done